### DISCUSSION TITLE
OK to upgrade to 3.0.2?

### DISCUSSION THREAD

**Discussion by drjjr2** [zwave-js/zwave-js-ui#955] [Q&A]
This might be a dumb question but do I need to wait for the April Home Assistant upgrade to update my zwavejs2mqtt? I am currently running 2.4.1 on HA 2021.3.4 but I have read a note somewhere that zwavejs 7.x needs HA 2021.4.x.

**Comment by robertsLando** [comment]
It is shipped with zwavejs server 1.3.0 that shouild have a back compatibility with older hass versions so yes should be safe but I didin't tried it

**Reply by drjjr2** [reply]
Excellent...Here's goes nothing.

**Reply by drjjr2** [reply]
After two days I would say that it seems to be working. I do not get alarmtype notifications for my Kwikset 910 locks. But now the lock states are correct so that's not as important to me as it used to be.
